src/app/InteractionModelEngine.cpp

```diff
@@ -1813,6 +1813,7 @@ Protocols::InteractionModel::Status InteractionModelEngine::CheckCommandFlags(co
 {
     const bool commandNeedsTimedInvoke = entry.flags.Has(DataModel::CommandQualityFlags::kTimed);
     const bool commandIsFabricScoped   = entry.flags.Has(DataModel::CommandQualityFlags::kFabricScoped);
+    const bool commandHasLargePayload  = entry.flags.Has(DataModel::CommandQualityFlags::kLargeMessage);
```

Comment:
Let's start moving code where it is needed. Having these constants at the top just makes us look back and forth between code.

The code below should read:

```cpp

if (entry.flags.Has(DataModel::CommandQualityFlags::kLargeMessage) &&
    !CurrentExchange()->GetSessionHandle()->AllowsLargePayload())
{
    return Status::InvalidAction;
}

// or

VerifyOrReturnError(
   !entry.flags.Has(DataModel::CommandQualityFlags::kLargeMessage) ||
       CurrentExchange()->GetSessionHandle()->AllowsLargePayload(),
   Status::InvalidAction
);
```